hack/kcp-edge.sh

```diff
@@ -0,0 +1,253 @@
+            kubectl apply -f https://raw.githubusercontent.com/kcp-dev/edge-mc/main/config/crds/meta.kcp.io_apiresources.yaml &> /dev/null
+
+            # Apply Exports
+            kubectl apply -f https://raw.githubusercontent.com/kcp-dev/edge-mc/main/config/exports/apiexport-edge.kcp.io.yaml &> /dev/null
+            kubectl apply -f https://raw.githubusercontent.com/kcp-dev/edge-mc/main/config/exports/apiexport-meta.kcp.io.yaml &> /dev/null
+            kubectl apply -f https://raw.githubusercontent.com/kcp-dev/edge-mc/main/config/exports/apiresourceschema-apiresources.meta.kcp.io.yaml &> /dev/null
+            kubectl apply -f https://raw.githubusercontent.com/kcp-dev/edge-mc/main/config/exports/apiresourceschema-customizers.edge.kcp.io.yaml  &> /dev/null
+            kubectl apply -f https://raw.githubusercontent.com/kcp-dev/edge-mc/main/config/exports/apiresourceschema-edgeplacements.edge.kcp.io.yaml &> /dev/null
+            kubectl apply -f https://raw.githubusercontent.com/kcp-dev/edge-mc/main/config/exports/apiresourceschema-edgesyncconfigs.edge.kcp.io.yaml &> /dev/null
+            kubectl apply -f https://raw.githubusercontent.com/kcp-dev/edge-mc/main/config/exports/apiresourceschema-singleplacementslices.edge.kcp.io.yaml &> /dev/null
+            kubectl apply -f https://raw.githubusercontent.com/kcp-dev/edge-mc/main/config/exports/apiresourceschema-syncerconfigs.edge.kcp.io.yaml &> /dev/null
+            echo "Finished populate the espw with kcp edge crds and apiexports"
+        else
+            echo "Unknown user type ..."
+            exit 1
+        fi 
+   fi
+fi
+
+sleep 5
+
+mailbox-controller --inventory-context=root --mbws-context=base -v=2 >& mailbox-controller-log.txt &
+
+run_status=$(wait_for_process mailbox-controller)
+if [ $run_status -eq 0 ]; then
+    echo " mailbox-controller is running (log file: mailbox-controller-log.txt)"
+else
+    echo " mailbox-controller failed to start ..... exiting"
+    sleep 2
+    exit
+fi
+
+
+# Start the edge-mc scheduler
+sleep 3
+scheduler -v 2 --root-user kcp-admin  --root-cluster root  --sysadm-context system:admin  --sysadm-user shard-admin >& edge-scheduler-log.txt &
+
+run_status=$(wait_for_process "scheduler -v 2")
+if [ $run_status -eq 0 ]; then
+    echo " scheduler is running (log file: edge-scheduler-log.txt)"
+else
+    echo " scheduler failed to start ..... exiting"
+    exit
+fi
+ 
+# Start the Placement Translator
+sleep 3
+placement-translator --allclusters-context  "system:admin" -v=2 >& placement-translator-log.txt &
+
+run_status=$(wait_for_process placement-translator)
+if [ $run_status -eq 0 ]; then
+    echo " placement translator is running (log file: placement-translator-log.txt)"
+else
+    echo " placement translator failed to start ..... exiting"
+    exit
+fi
+
+sleep 10
+echo "****************************************"
+echo "Finished deploying kCP-EDGE controllers ...."
+echo "****************************************"
+kubectl ws root
```

Comment:
What is this for?

Reply:
Just to return to the root workspace after deploying kubestellar - this was also one of the feedbacks I received from Andy

Reply:
OK